src/sentry/web/frontend/organization_settings.py

```diff
@@ -57,17 +90,32 @@ def get_form(self, request, organization):
                 'allow_joinleave': bool(organization.flags.allow_joinleave),
                 'enhanced_privacy': bool(organization.flags.enhanced_privacy),
                 'allow_shared_issues': bool(not organization.flags.disable_shared_issues),
+                'scrub_data': bool(organization.get_option('sentry:scrub_data', False)),
```

Comment:
this actually needs to change for org settings. We probably need a select box which is:

- Enabled for all Projects
- Let each project decide